src/pages/Trade/Trade.js

```diff
@@ -19,6 +20,7 @@ const Trade = () => {
 		<MainLayout>
 			<WalletPopup />
 			<GweiPopup />
+			<Banner />
```

Comment:
The `<Banner />` component should be in the `Root` component

Reply:
yeah I got rid of it